docs/src/pipe.rst

```diff
@@ -113,3 +145,17 @@ API
     function is blocking.
 
     .. versionadded:: 1.16.0
+
+.. c:function:: int uv_pipe_get_credentials(uv_pipe_t* handle, uv_credentials_t* creds)
+
+    Retrieves the remote client's PID and effective UID/GID.
+
+    On Windows, only the ``pid`` field of the credentials object is populated; ``uid``
+    and ``gid`` are set to ``-1``.
+
+    .. note::
+        This information is considered secure, as it cannot be spoofed by the connecting
+        process. On non-Windows systems, the UID and GID can be used to safely enforce access
+        control to a listening pipe, for example.
+
+    .. versionadded:: 1.29.0
```

Comment:
Wasn't sure if this should be added or not; I took a guess here.

Reply:
Please set it to 1.32.0